### ISSUE TITLE
Sense of doom

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6784]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is a manifestation of Anxiety and should not be a separate term. Adding as a narrow synonym to Anxiety
